provers/evm-prover/src/main.rs

```diff
@@ -85,6 +86,9 @@ impl ProverService {
 
         let genesis = Genesis::Custom(genesis_json);
 
+        let client = ProverClient::from_env();
```

Comment:
ideally we could pass client around instead of throwing it away and recreating it in `prover.rs`. Not a blocker, just an idea for simplifying later.

Reply:
updated